webpack.config.js

```diff
@@ -79,15 +79,14 @@ const libraryUMDConfig = merge([
     {
         output: {
             path: PATHS.umd,
-            filename: '[name].min.js',
+            filename: '[name].js',
```

Comment:
You've probably already thought this through, but I just wanted to ask. Do we want to remove the 'min' part from the filename? I think we already have users and this will likely be a breaking change for them. Also, to maintain consistency with the UMD builds in britecharts which I believe also have .min.js extensions.

Reply:
Great question!
So it seems that whatever we put in /lib is supposed to be used within the apps of our users, so they should minify it by themselves.
What I am doubting we can do is, as we do in britecharts, include individual bundles on the /dist folder. They should be consistent, that's for sure, so maybe we could do:
```
/dist (minified)
   /bundle
   /umd
/lib (all non-minified)
   /umd
   /cjs
   /esm
```
This way we would offer a good amount of options to our users. 
What do you think?

Reply:
Oh I see, that makes sense. And you wouldn't want to name it min if it won't be minified. It makes sense that users should minify it themselves.

Actually this got me interested in googling to see if anyone's written best practices for npm library authors, and I couldn't find much. But I noticed that NPM actually discourages you from bundling UMD builds: https://www.npmjs.com/package/module-best-practices#umd-builds

It might make sense to provide the UMD versions through CDN only, and leave it out of the NPM package. Since you should only use the UMD one if you're using it as a <script> tag, and in that case you wouldn't be installing through NPM. But the way you mentioned is easier, if we have to publish to a CDN with each build then we have to set up automation for that.

Reply:
I didn't see the part of not minimizing it.
Yeah, there is little on the internet about library development. Ben did a lot of research and he is a great resource in this. 
Oh, we have the CDN already working for both projects: https://www.jsdelivr.com/package/npm/britecharts
It's automatic!